### DISCUSSION TITLE
Can you mirror a repository and preserve pull requests and issues

### DISCUSSION THREAD

**Discussion by wehimwich** [community/community#27283] [Other Feature Feedback, Questions, & Ideas]
The instructions for mirroring a repository at [duplicating-a-repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/duplicating-a-repository) worked great. Is it possible to also preserve the pull requests and issues (and maybe other parts of the environment)? Part of the point of using mirroring was to preserve the previous history in a new project. The instructions referenced above worked great for the code, but we lost the other parts of the history. A small example of this is that all the `#` references in the commits and docs now refer to non-existent, and what will eventually be different, issues/PRs/discussions.

**Comment by jfullstackdev** [comment]
maybe this one is similar and might help

https://stackoverflow.com/questions/54745614/git-mirror-with-issues-and-pull-request-history

**Reply by wehimwich** [reply]
Thanks for the info. It is interesting in its own right, but not quite what I am looking for. I think there would need to be a GitHub feature that would be like a transferring ownership, but would let you make a duplicate under the same owner.

**Comment by kaiserkiwi** [comment]
Did you find a solution for this? I'm looking for the same thing.

**Comment by souraaav** [comment]
yes you can 

first - use github api .
second - clone the repository to your pc .
third - use git hub api to create new pull request and issue in the new repo coping over the details from the original
fourth - you need to push all the branches and tags to the new repo .
note - this process might be simpler if the repo is transferrd to a different owner within github, as that preserves most of the data 

hope so that helps
